quickstart/armv7m7-imxrt117x-evk.md

```diff
@@ -54,17 +62,34 @@ that run:
 (in this case ACM0)
 
   ```bash
-  picocom -b 115200 --imap lfcrlf /dev/ttyACM0
+  picocom -b 115200 --imap lfcrlf /dev/tty[port]
   ```
 
   <details>
-  <summary>How to get picocom (Ubuntu 20.04)</summary>
+  <summary>How to get picocom and run it without privileges (Ubuntu 22.04)</summary>
 
   ```bash
   sudo apt-get update && \
   sudo apt-get install picocom
   ```
 
+ To use picocom without sudo privileges run this command and then restart:
```

Comment:
`To use picocom without sudo...` should be within the dropdown list:
![Screenshot from 2023-12-14 20-02-51](https://github.com/phoenix-rtos/phoenix-rtos-doc/assets/77304431/1891d38b-27d4-4246-a02b-a777a1e9bc3f)